test/dev-server.unit.js

```diff
@@ -200,3 +200,14 @@ test(
     }
   })
 );
+
+test(
+  '[DevServer] Test @now/static-build routing',
+  testFixture('now-dev-static-build-routing', async (t, server) => {
+    {
+      const res = await fetch(`${server.address}/api/date`);
```

Comment:
Since this fixture has a `now.json` file (so not zero-config), the URL needs to have the .js suffix.

```suggestion
      const res = await fetch(`${server.address}/api/date.js`);
```

Reply:
@TooTallNate I've added a route to the test

 ```
  "routes": [
    { "src": "^/api/date$", "dest": "api/date.js" }
  ]
```